templates/charter/wg-charter-template.md

```diff
@@ -0,0 +1,40 @@
+# WG #WGNAME Charter
+
+| Status        | (Proposed / Accepted / Rejected / Archived)       |
+:-------------- |:---------------------------------------------------- |
+| **PR Link**     | [PR #](https://github.com/icon-project/community/pulls/#) (update when you have PR link)|
+| **Author(s)** | My Name (me@example.org), AN Other (you@example.org) |
+| **Sponsor**   | Steering committee member name (example@icon.foundation)                 |
+| **Updated**   | DD MONTH YYYY                                           |
+
+This charter adheres to the [WG governance guidelines](/guidelines/governance/wg-governance-guidelines.md).
+
+## Overview
+
+Briefly explain to the community why your WG could matter to them.
+
+## Relationship to SIGs
+
+Describe 
+
+## Scope
+
+Describe all technical and non-technical components within the scope of the WG.
+
+### Out of scope
+
+Clarify anything that could be considered to be in scope but are not.
+
+## Governance
+
+This WG adheres to the [WG governance guidelines](/guidelines/governance/wg-governance-guidelines.md). Any changes from the guidelines should be noted here.
+
+### Roles and Responsibilities
+
+Briefly describe the roles and responsibilities of various members of the WG that are not defined in the [WG governance guidelines](/guidelines/governance/wg-governance-guidelines.md).
+
+## Disbanding
```

Comment:
Disbanding is already mentioned in governance. Why not just link to that section?